docs/ide-usage.md

```diff
@@ -53,6 +53,14 @@ The following plugins are needed:
      * ORM Enhancement: `App Engine → ORM` → clear all the entries, and add the following entry: `src/main/java/teammates/storage/entity/*.java`.
      * Validation exclusion: `App Engine → Validation` → add two entries: `src/test/java` and `src/client/java`.
      * WAR directory: `Web Application` → tick both `This project has a WAR directory` and `Launch and deploy from this directory`, and enter `src/main/webapp` as `WAR directory`.
+   * Java Compiler: Go to `Project → Properties → Java Compiler` and  tick `"Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'" `.
```

Comment:
* JDK: `Java Build Path → Libraries` → ensure that the system library used is JDK 7.
* Compiler compliance: `Java Compiler` → tick `Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'`

The order needs to be that way as the build path needs to be set first before compiler compliance.

Also, the image for compiler compliance looks off (different resolution from the rest). Use this instead:
![screen shot 2017-04-14 at 9 48 13 pm](https://cloud.githubusercontent.com/assets/7261051/25050293/340984ec-2179-11e7-87ab-d472255a2cf9.png)